config/optional/block.block.nodeheaderblock.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - localgov_core
+  theme:
+    - localgov_theme
```

Comment:
I wonder if localgov_theme needs to go into the info file and composer.json of this module.  Something to think about.

Reply:
On a second thought, given this is an "optional" config file, the module info and composer.json files can skip requiring the localgov_theme.